lib/phoenix_live_view/helpers.ex

```diff
@@ -306,4 +306,34 @@ defmodule Phoenix.LiveView.Helpers do
 
     EEx.compile_string(expr, options)
   end
+
+  @doc """
+  Renders a title tag with automatic prefix/postfix on `@page_title` updates.
+
+  ## Examples
+
+      <%= live_title_tag @page_title, prefix: "MyApp – " %>
+
+      <%= live_title_tag @page_title, postfix: " – MyApp" %>
```

Comment:
This is a very minor nitpick but the String module uses `prefix` and `suffix`. I am not sure if you want to standardize on it.